README.md

```diff
@@ -49,7 +49,11 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
    - auth-url
    - password
 
-   You can get those information from your cloud provider.
+   You can get this information from your cloud provider. You can also download the clouds.yaml from your cloud provider and  pass it to the generator. **Note: Make sure your clouds.yaml file has all the fields listed above, otherwise clusterctl will fail. You will not be prompted for input if you pass the generator a clouds.yaml file. Note: the default cloud is openstack.** It can be passed to the generator with the -c or --clouds command like so:
+   
```

Comment:
Actually, can you also format this paragraph a bit better? Separate the notes from the rest of the paragraph, for example. That should make it more readable. :smile: